airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/CdcAcceptanceTests.java

```diff
@@ -0,0 +1,268 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class CdcAcceptanceTests {
+
+  record DestinationCdcRecordMatcher(JsonNode sourceRecord, Instant minUpdatedAt, Optional<Instant> minDeletedAt) {}
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(BasicAcceptanceTests.class);
+
+  private static final String POSTGRES_INIT_SQL_FILE = "postgres_init_cdc.sql";
+  // must match replication slot name used in the above POSTGRES_INIT_SQL_FILE
+  private static final String REPLICATION_SLOT = "airbyte_slot";
+  // must match publication name used in the above POSTGRES_INIT_SQL_FILE
+  private static final String PUBLICATION = "airbyte_publication";
+
+  private static final String SCHEMA_NAME = "public";
+  private static final String CDC_UPDATED_AT_COLUMN = "_ab_cdc_updated_at";
+  private static final String CDC_DELETED_AT_COLUMN = "_ab_cdc_deleted_at";
+
+  private static AirbyteAcceptanceTestHarness testHarness;
+  private static AirbyteApiClient apiClient;
+  private static UUID workspaceId;
+  private static PostgreSQLContainer sourcePsql;
+
+  @BeforeAll
+  public static void init() throws URISyntaxException, IOException, InterruptedException, ApiException {
+    apiClient = new AirbyteApiClient(
+        new ApiClient().setScheme("http")
+            .setHost("localhost")
+            .setPort(8001)
+            .setBasePath("/api"));
+    // work in whatever default workspace is present.
+    workspaceId = apiClient.getWorkspaceApi().listWorkspaces().getWorkspaces().get(0).getWorkspaceId();
+    LOGGER.info("workspaceId = " + workspaceId);
+
+    // log which connectors are being used.
+    final SourceDefinitionRead sourceDef = apiClient.getSourceDefinitionApi()
+        .getSourceDefinition(new SourceDefinitionIdRequestBody()
+            .sourceDefinitionId(UUID.fromString("decd338e-5647-4c0b-adf4-da0e75f5a750")));
+    final DestinationDefinitionRead destinationDef = apiClient.getDestinationDefinitionApi()
+        .getDestinationDefinition(new DestinationDefinitionIdRequestBody()
+            .destinationDefinitionId(UUID.fromString("25c5221d-dce2-4163-ade9-739ef790f503")));
+    LOGGER.info("pg source definition: {}", sourceDef.getDockerImageTag());
+    LOGGER.info("pg destination definition: {}", destinationDef.getDockerImageTag());
+
+    testHarness = new AirbyteAcceptanceTestHarness(apiClient, workspaceId, POSTGRES_INIT_SQL_FILE);
+    sourcePsql = testHarness.getSourcePsql();
+  }
+
+  @AfterAll
+  public static void end() {
+    testHarness.stopDbAndContainers();
+  }
+
+  @BeforeEach
+  public void setup() throws SQLException, URISyntaxException, IOException {
+    testHarness.setup();
+  }
+
+  @AfterEach
+  public void tearDown() {
+    testHarness.cleanup();
+  }
+
+  @Test
+  @Order(12)
```

Comment:
I think that you can just remove all of the `Order` annotations.  My understanding is that each of these tests is self-contained and doesn't require another test to set up state first before it can run.  Therefore, we don't need to stipulate an order.